### PR TITLE
Return connection in attach method

### PR DESCRIPTION
**Pull request by Maxpain177**
For the ability to close WebSocket connections.

### CODE REVIEW

File: src/attach.ts [hunk 1]
```diff
@@ -34,5 +34,7 @@ export class Attach {
         if (stdin != null) {
             WebSocketHandler.handleStandardInput(conn, stdin);
         }
+
+        return conn
```
[the remark this answers is not in the document]
Comment (Maxpain177) [reply src/attach.ts]: done

File: src/attach.ts [hunk 2]
```diff
@@ -34,5 +35,7 @@ export class Attach {
...
```
[the remark this answers is not in the document]
Comment (itowlson) [reply src/attach.ts]: Oh, we have got a lint rule for it, and that's why CI is failing *grin*

**Comment by drubin** [comment]
/lgtm

It would be nice to squash the commits when merging though. @brendanburns is that possible?

**Comment by itowlson** [comment]
@drubin Yes, it is - the `Merge pull request` button in the GitHub UI has a drop-down which you can use to change the option to `Squash and merge`.

**Comment by drubin** [comment]
Yes but the bot does the merging. So wanted to confirm the correct proccess.

**Comment by itowlson** [comment]
Doh!  You are right of course @drubin!  Too much manual maintenance, don't have the k8s process ingrained properly.

**Comment by brendandburns** [comment]
I manually squash/merged this. I'll see if we can setup the bot config to do that in the future.

**Pull request merged**